operator/pkg/model/translation/cec_translator.go

```diff
@@ -133,16 +133,16 @@ func (i *cecTranslator) getBackendServices(m *model.Model) []*ciliumv2.Service {
 	return res
 }
 
-func (i *cecTranslator) getServicesWithPorts(namespace string, name string, model *model.Model) []*ciliumv2.ServiceListener {
-	// Find all the ports used in the model and build a set of them
+func (i *cecTranslator) getServicesWithPorts(namespace string, name string, m *model.Model) []*ciliumv2.ServiceListener {
+	// Find all the ports used in the m and build a set of them
```

Comment:
search and replace was too greedy here, I think.

Reply:
oops thanks a lot.